.travis.yml

```
@@ -6,26 +6,32 @@ services:
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.3
       env: SYMFONY_VERSION="3.4.*" COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.2
-      env: SYMFONY_VERSION="4.4.*"
     - php: 7.3
       env: SYMFONY_VERSION="4.4.*"
     - php: 7.4
       env: SYMFONY_VERSION="4.4.*"
+    - php: 8.0
+      env: SYMFONY_VERSION="4.4.*"
     - php: 7.4
       env: SYMFONY_VERSION="5.0.*"
-    - php: 7.2
-      env: SYMFONY_VERSION="5.1.*"
+    - php: 8.0
+      env: SYMFONY_VERSION="5.0.*"
     - php: 7.4
       env: SYMFONY_VERSION="5.1.*"
+    - php: 8.0
+      env: SYMFONY_VERSION="5.1.*"
     - php: 7.4
       env: SYMFONY_VERSION="5.2.*" STABILITY="dev"
+    - php: 8.0
+      env: SYMFONY_VERSION="5.2.*" STABILITY="dev"
```

Symfony 5.2 is now stable.

```suggestion
      env: SYMFONY_VERSION="5.2.*"
```